package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "axios": "^1.6.8",
-    "next": "14.2.1",
+    "next": "^14.2.5",
+    "node": "^22.6.0",
```

Comment:
This seems to be an unintentional effect of you attempting to troubleshoot... we do not want to increase our next version or node version with this PR